sdk/scheduler/src/main/java/com/mesosphere/sdk/scheduler/SchedulerConfig.java

```diff
@@ -387,7 +387,7 @@ public int getStatsdPort() {
      * Returns the Mesos API version.
      */
     public String getMesosApiVersion() {
-        return envStore.getRequired(MESOS_API_VERSION_ENV);
+        return envStore.getOptional(MESOS_API_VERSION_ENV, "V1");
```

Comment:
This means that we *could* still be operating with `V0` even though we are removing the tests for this. Is there a way to *ensure* that we always use V1 -- even for existing installations after an update? cc @nickbp.

That is to say, if a customer is running a service using `V0`, and performs an update to a scheduler using `V1`, is there a reasonable update path?

Reply:
yes this is intentional, this is meant as an escape hatch for anyone who needs to use V0, they can manually add the envar. AFAIK update from v0 to v1 should be seamless.

Reply:
With the removal of the api version option in config.json, I think in practice the customer would see that the option was no longer valid when upgrading, and would then remove it. As a result they would effectively be switched over to v1 when the new scheduler starts.

To be clear I don't expect anybody to be using the v0 option, or at least not for any good reason. It was only originally added as a fallback if we had problems with migrating to v1